src/main/resources/languages/english.yml

```diff
@@ -35,6 +35,18 @@ Items:
     Name: "Energy Drink"
   Handcuffs:
     Name: "Handcuffs"
+  PlasticSpoon:
+    Name: "Wooden Shovel"
```

Comment:
There might be a typo here. Did u mean Wooden Spoon?

Reply:
It is supposed. Since we are using weaker shovels to represent spoons I think it will be good to name it shovel as well. This way players won't be confused. Either way that is just the config name so it can easily be changed later.